build-info-extractor-go/src/test/java/org/jfrog/build/extractor/go/GoDriverTest.java

```diff
@@ -53,4 +56,27 @@ public void testListUsedModules() throws IOException {
             FileUtils.deleteDirectory(projectDir);
         }
     }
+
+    @Test
+    public void testTidyListUsedModulesErroneous() throws IOException {
```

Comment:
Let's add a descxription for this test method, and also a description for the above ```testTidyListUsedModules``` method.